src/ext-test/java/org/opentripplanner/ext/vectortiles/layers/areastops/AreaStopsLayerBuilderTest.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.ext.vectortiles.layers.areastops;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import java.util.Locale;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner._support.geometry.Polygons;
+import org.opentripplanner.ext.vectortiles.VectorTilesResource;
+import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.inspector.vector.LayerParameters;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.site.AreaStop;
+import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.StopModel;
+import org.opentripplanner.transit.service.StopModelBuilder;
+import org.opentripplanner.transit.service.TransitModel;
+
+class AreaStopsLayerBuilderTest {
+
+  private static final FeedScopedId ID = new FeedScopedId("FEED", "ID");
+  private static final I18NString NAME = I18NString.of("Test stop");
+
+  private final StopModelBuilder stopModelBuilder = StopModel.of();
+
+  private final AreaStop AREA_STOP = stopModelBuilder
+    .areaStop(ID)
+    .withName(NAME)
+    .withGeometry(Polygons.BERLIN)
+    .build();
+
+  private final TransitModel transitModel = new TransitModel(
+    stopModelBuilder.withAreaStop(AREA_STOP).build(),
+    new Deduplicator()
+  );
+
+  record Layer(
+    String name,
+    VectorTilesResource.LayerType type,
+    String mapper,
+    int maxZoom,
+    int minZoom,
+    int cacheMaxSeconds,
+    double expansionFactor
+  )
+    implements LayerParameters<VectorTilesResource.LayerType> {}
+
+  @Test
+  void getAreaStops() {
+    transitModel.index();
+
+    var layer = new Layer(
+      "areaStops",
+      VectorTilesResource.LayerType.AreaStop,
+      "OTPRR",
+      20,
+      1,
+      10,
+      .25
+    );
```

Comment:
I sort of prefer the approach where we use a proper config. For example https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/ext-test/java/org/opentripplanner/ext/vectortiles/layers/vehicleparkings/VehicleParkingsLayerTest.java#L93

Reply:
Fixed in https://github.com/ibi-group/OpenTripPlanner/commit/0136bbd59b3f5fd24cfaea9c90d448de6b4c8b12